packages/terra-dropdown-button/src/Item.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import styles from './Item.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
```

Comment:
@neilpfeiffer, should the drop down items have a 'selected' option that adds a checkmark for toggled items? I only ask because thats what we ended up doing in terra-dev-site for the toolbar buttons.

Reply:
@mjhenkes No - the list of items in the Dropdown and Splitbuttons are straight buttons, not toggles, so "selecting" or "pressing" them immediately would do an action. There is no persistent "selected" state for these buttons or the items in the list. That would be a menu or other component. This is "pick and execute".